.github/workflows/manual-release-to-npm.yml

```diff
@@ -20,19 +20,16 @@ jobs:
           NPM_EMAIL: ${{secrets.NPM_EMAIL}}
           NPM_USERNAME: ${{secrets.NPM_USERNAME}}
         run: yarn release maintenance
-  tag:
-    needs: release
-    if: "startsWith(github.event.head_commit.message, 'chore(release)')"
-    runs-on: ubuntu-latest
-    name: Tag release commit
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
+      - name: Get commit message
+        run: |
+          echo "commmit_message=$(git log --format=%B -n 1 ${{ github.event.after }})" >> $GITHUB_ENV
       - name: Set up git identity
+        if: "startsWith(env.commmit_message, 'chore(release)')"
         run: git config --global user.name "instructure-ui-ci" && git config --global user.email "instructure-ui-ci@instructure.com"
         # -a -m is needed to make the tag "annotated". Lerna needs annotated tags to work correctly with verisons
       - name: Add tag
+        if: "startsWith(env.commmit_message, 'chore(release)')"
         run: git tag -a v$(./.github/workflows/calculateVersion.sh) -m v$(./.github/workflows/calculateVersion.sh)
       - name: Push tags
+        if: "startsWith(env.commmit_message, 'chore(release)')"
         run: git push origin v$(./.github/workflows/calculateVersion.sh)
```

Comment:
Cant we just simply merge these steps into 1 step?

Reply:
Not really. The git cli works weirdly with chained commands. Besides, much better to break it up into smaller steps to see where a problem occurs when it inevitably does